lib/db/models.py

```diff
@@ -28,6 +28,11 @@ class Comment(BaseModel):
 
 class Issue(BaseModel):
     issue_id = pw.IntegerField(primary_key=True)
+    number = pw.IntegerField()
```

Comment:
Hmm... interesting. Most of the API I saw for issues and PRs take the issue/PR number, though?